Ginger/Ginger/DragDropLib/DragDrop2.cs

```diff
@@ -29,14 +30,14 @@ public static class DragDrop2
     {
         private static Point _startPoint;
         private static bool IsDragging;
-        public static DragInfo DragInfo;
+        public static DragInfo dragInfo;
```

Comment:
you removed the 'm' from 'dragInfo' but forgot to return it also to start with Captial D to be "DragInfo"